### PR TITLE
Feature/chinedu/user request service

### PR DESCRIPTION
**Pull request by ChineduOguejiofor**
#### What does this PR achieve?
- Connected the Order to the backend
- Added Order summary Modal
- Rearranged UI for making an order

#### Screenshots:
rrr

#### Relevant Trello Tickets
- Order Request User

### CODE REVIEW

File: src/components/userRequestService/components/Info.jsx [hunk 1]
```diff
@@ -73,6 +66,7 @@ const Info = () => {
             <RaidoComp onChange={onChange} />
             <CheckBox onCheck={onCheck} />
             <TextAreaAndButton onChange={onChange} info={info} />
+
```
Comment (Way2nnadi) [inline src/components/userRequestService/components/Info.jsx]: remove unused space, make sure you have your eslint installed in VS code

File: src/components/userRequestService/components/ItemsTable.jsx [hunk 2]
```diff
@@ -1,61 +1,53 @@
+/* eslint-disable max-lines-per-function */
```
Comment (Way2nnadi) [inline src/components/userRequestService/components/ItemsTable.jsx]: Remove this display

File: src/components/userRequestService/components/TablesDisplay.jsx [hunk 3]
```diff
@@ -1,46 +1,57 @@
+/* eslint-disable react/jsx-fragments */
```
Comment (Way2nnadi) [inline src/components/userRequestService/components/TablesDisplay.jsx]: You can remove this if you enable `eslint-plugin-react`, if you currently have it make sure to update it and be using at least React v16.2

File: src/components/userRequestService/components/TablesDisplay.jsx [hunk 4]
```diff
@@ -1,46 +1,57 @@
...
+    return (
+        <Col xs={13} sm={16} md={24} lg={22} xl={{ offset: 0, span: 14 }} xxl={12}>
+            <Card title={SPECIFY} className="border">
+                <ItemsTable />
+                <Divider />
+                <>
+                    <p>
+                        {totalPrice}
+                    </p>
+                </>
```
Comment (Way2nnadi) [inline src/components/userRequestService/components/TablesDisplay.jsx]: no need for the fragment here

File: src/components/userRequestService/components/TablesDisplay.jsx [hunk 5]
```diff
@@ -1,46 +1,57 @@
...
+                <Info />
+            </Card>
+        </Col>
+    );
+};
 
 const DisplayRow = () => (
     <Row gutter={DisplayRowStyle}>
-        <ItemsAndInfo />
+
```
Comment (Way2nnadi) [inline src/components/userRequestService/components/TablesDisplay.jsx]: remove unused space

File: src/components/userRequestService/components/TablesDisplay.jsx [hunk 6]
```diff
@@ -1,46 +1,57 @@
...
 const TablesDisplay = () => (
+
     <Card className="border-radius">
         <Title level={4}>
             {REQUEST}
         </Title>
         <Divider />
         <DisplayRow />
     </Card>
+
```
Comment (Way2nnadi) [inline src/components/userRequestService/components/TablesDisplay.jsx]: come on chinedu, this is just basic, go through your PR and catch this minor mistakes and fix them so reviewers dont have to make comments on minor things like this

File: src/components/userRequestService/reducer.js [hunk 7]
```diff
@@ -2,42 +2,54 @@
...
-            current: null,
             dataSource:
-            state.dataSource.map(record => (record.key === payload.key ? payload : record)),
+            state.dataSource.map(record => (record.key === payload.key
+                ? { ...record, quantity: payload.value } : record)),
+            totalPrice: state.dataSource.reduce(
+                (accumalatedQuantity, record) => accumalatedQuantity
+                + record.quantity * record.price,
+                0
+            ),
```
Comment (Way2nnadi) [inline src/components/userRequestService/reducer.js]: this is confusing, abstract this out into a function, for the most part your reducers are use updating the stating. It should not have any additional logic

File: src/components/userRequestService/components/Info.jsx (hunk 1 above)
Comment (ChineduOguejiofor) [reply src/components/userRequestService/components/Info.jsx]: Removed

File: src/components/userRequestService/components/ItemsTable.jsx (hunk 2 above)
Comment (ChineduOguejiofor) [reply src/components/userRequestService/components/ItemsTable.jsx]: I actually forgot to remove this
Sorry about that
Thanks

File: src/components/userRequestService/components/TablesDisplay.jsx (hunk 3 above)
Comment (ChineduOguejiofor) [reply src/components/userRequestService/components/TablesDisplay.jsx]: I don't know how to achieve this

File: src/components/userRequestService/components/TablesDisplay.jsx (hunk 4 above)
Comment (ChineduOguejiofor) [reply src/components/userRequestService/components/TablesDisplay.jsx]: Removed

File: src/components/userRequestService/components/TablesDisplay.jsx (hunk 5 above)
Comment (ChineduOguejiofor) [reply src/components/userRequestService/components/TablesDisplay.jsx]: removed

File: src/components/userRequestService/components/TablesDisplay.jsx (hunk 6 above)
Comment (ChineduOguejiofor) [reply src/components/userRequestService/components/TablesDisplay.jsx]: I apologize for this,
I would be more careful subsequently.
Thanks

Comment (ChineduOguejiofor) [reply src/components/userRequestService/components/TablesDisplay.jsx]: I apologize for this,
I would be more careful subsequently.
Thanks

File: src/components/userRequestService/reducer.js (hunk 7 above)
Comment (ChineduOguejiofor) [reply src/components/userRequestService/reducer.js]: I have abstracted this into a utils.js file

File: src/components/userRequestService/components/TablesDisplay.jsx (hunk 3 above)
Comment (ChineduOguejiofor) [reply src/components/userRequestService/components/TablesDisplay.jsx]: I will look into it a bit more later

**Pull request merged**